src/core/NewDomain.ml

```diff
@@ -662,14 +724,14 @@ struct
 
     | Neu info ->
       let neu = {head = NCoe {r; r'; ty = Abs (x, info.neu); cap}; frames = Emp} in
-      let ty = Val.subst_then_run (Rel.hide' x rel) r x tyx in
+      let ty = Delayed.make' (Some (Rel.hide' x rel)) (subst r' x tyx) in
```

Comment:
I think the previous version, which was trying to substitution `<r/x>`, is wrong. It should be `r'/x`.

Reply:
Never mind, this was introduced by me in ead41d1a15030aa1ad3dda94d185a6f7204e10ab LOL.